projects/sitemanage/src/test/java/com/percussion/share/test/PSMatchers.java

```diff
@@ -104,14 +104,22 @@ public static Matcher<String> validXhtml() {
             private Collection<SAXParseException> errors = new ArrayList<SAXParseException>();
             
             @Override
-            public boolean matchesSafely(String item) throws UnsupportedCharsetException
+            public boolean matchesSafely(String item)
             {
                 PSXhtmlValidator validator = new PSXhtmlValidator();
                 InputStream stream;
-                stream = IOUtils.toInputStream(item, "UTF-8");
+                try
+                {
+                    stream = IOUtils.toInputStream(item, "UTF-8");
+                }
+                catch (UnsupportedCharsetException e)
+                {
+                    throw new UnsupportedCharsetException("Exception in method matchesSafely(): "+e.getMessage());
+                }
                 boolean valid = validator.isValid(stream);
```

Comment:
@natechadwick Yes Nate you are right but since its a part of test classes, that's why throwing it so that it fails in the test case itself. The way it was written before.
If you think that we should just catch it and return false, then that also we can do.

Reply:
@natechadwick I have done the changes as suggested.